queenbee/workflow/status.py

```diff
@@ -76,12 +76,12 @@ class TaskStatus(BaseStatus):
         description='The command used to run this task. Only applies to Function tasks.'
     )
 
-    inputs: Arguments = Field(
+    inputs: TaskArguments = Field(
         ...,
         description='The inputs used by this task'
     )
 
-    outputs: Arguments = Field(
+    outputs: TaskReturns = Field(
```

Comment:
I'm not sure if this change is the right change but since this is a task I feel the inputs and outputs should be TaskIO.

Reply:
I can't find a class called `TaskIO`. What should it look like and how would it differ from `TaskArguments` + `TaskReturns`?